src/bitbox02.js

```diff
@@ -89,6 +89,20 @@ export class BitBox02API {
     constructor(devicePath)  {
         this.devicePath = devicePath;
         this.opened = false;
+        this.socket = null;
+    }
+
+    connectWebsocket() {
+        const socket = new WebSocket("ws://127.0.0.1:8178/api/v1/socket/" + this.devicePath);
+        return new Promise((resolve, reject) => {
+            socket.binaryType = 'arraybuffer'
+            socket.onopen = async function (event) {
+                resolve(socket);
+            }
+            socket.onerror = function(event) {
+                reject("Your BitBox02 is busy");
+            }
```

Comment:
I think we don't use Automatic Semicolon Insertion (ASI), please add `;` here and after `socket.binaryType = 'arraybuffer'`